src/PDS4_RINGS_IngestLDD.xml

```diff
@@ -4077,54 +4079,18 @@
     </DD_Attribute>
 
     <DD_Attribute>
-        <name>reprojection_grid_radial_resolution_mean</name>
-        <version_id>1.0</version_id>
-        <local_identifier>reprojection_grid_radial_resolution_mean</local_identifier>
-        <nillable_flag>false</nillable_flag>
-        <submitter_name>M. Tiscareno</submitter_name>
-        <definition>reprojection_grid_radial_resolution_mean indicates <!--MORE NEEDED HERE-->
-            Required in labels.
-            Not intended to be used as a table field.
-        </definition>
-        <comment>
-            v1.0 (post-2020): Attribute creation
-        </comment>
-        <DD_Value_Domain>
-            <enumeration_flag>false</enumeration_flag>
-            <value_data_type>ASCII_Real</value_data_type>
-            <unit_of_measure_type>Units_of_Length</unit_of_measure_type>
-        </DD_Value_Domain>
-    </DD_Attribute>
-
-    <DD_Attribute>
-        <name>reprojection_grid_radial_resolution_min</name>
-        <version_id>1.0</version_id>
-        <local_identifier>reprojection_grid_radial_resolution_min</local_identifier>
-        <nillable_flag>false</nillable_flag>
-        <submitter_name>M. Tiscareno</submitter_name>
-        <definition>reprojection_grid_radial_resolution_min indicates <!--MORE NEEDED HERE-->
-            Required in labels.
-            Not intended to be used as a table field.
-        </definition>
-        <comment>
-            v1.0 (post-2020): Attribute creation
-        </comment>
-        <DD_Value_Domain>
-            <enumeration_flag>false</enumeration_flag>
-            <value_data_type>ASCII_Real</value_data_type>
-            <unit_of_measure_type>Units_of_Length</unit_of_measure_type>
-        </DD_Value_Domain>
-    </DD_Attribute>
-
-    <DD_Attribute>
-        <name>reprojection_grid_radial_resolution_max</name>
+        <name>reprojection_grid_radial_resolution</name>
         <version_id>1.0</version_id>
-        <local_identifier>reprojection_grid_radial_resolution_max</local_identifier>
+        <local_identifier>reprojection_grid_radial_resolution</local_identifier>

```

Comment:
Remove this attribute.  The only reprojection grid attributes in my final draft of issue 301 are reprojection_grid_[radial/longitudinal]_sampling_interval.